consumerui/Dockerfile

```diff
@@ -2,9 +2,10 @@ FROM ubuntu:20.04
 
 RUN apt-get update -y && DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata && apt-get install -y python3-pip python-setuptools  curl wget tar sudo apt-transport-https ca-certificates socat python-yaml vim graphviz
 
-RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-
-RUN cp /usr/bin/python3.8 /usr/bin/python
+RUN arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
```

Comment:
What is output of arch on a arm machine? Why do we need two sed commands?

Reply:
On Linux the arch command will return aarch64 for ARM  and x86_64 for AMD64 machine. Most ackage are published as either arm64 or amd64 so we need to run sed to translate. 
> Why do we need two sed commands?
Not the or (|)